extensions/resteasy-common/deployment/pom.xml

```diff
@@ -38,6 +38,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-resteasy-common</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-arc-deployment</artifactId>
+        </dependency>
```

Comment:
I talked a bit too fast. Could you also add the Arc runtime dependency to the runtime artifact (the one without deployment)?

Reply:
on it...

Reply:
updates done, single commit in the PR

Reply:
Hm, I wonder if it's actually needed. I mean `resteasy-common` does not depend on any class from the Arc runtime and `quarkus-arc-deployment` should bring the runtime in the app anyway. But maybe I'm missing something.

Reply:
main reason is to be able to use `BeanArchiveIndexBuildItem` to get the index out of it. Since there are generated rest endpoints that are cdi beans they exist only there.

Reply:
But `BeanArchiveIndexBuildItem` is a build-time construct and comes from the deployment module.

Reply:
so you're saying we don't need arc for runtime module, right?

Reply:
I guess so but as I said I might be missing something...

Reply:
I would prefer consistency as having one and not the other led to weird issues on quite a few extensions already.

Reply:
Ok np.